packages/dayjs/src/dayjs-utils.ts

```diff
@@ -207,8 +207,9 @@ export default class DayjsUtils implements IUtils<defaultDayjs.Dayjs> {
   }
 
   public getWeekdays() {
-    return [0, 1, 2, 3, 4, 5, 6].map(dayOfWeek =>
-      this.format(this.dayjs().set("day", dayOfWeek), "dd")
+    const start = this.dayjs().startOf("week");
+    return [0, 1, 2, 3, 4, 5, 6].map(diff =>
```

Comment:
I think in this case it will be better to work with good old `for (let i = 0; i < 7; i++)` Instead of clonning dates

Reply:
Generally speaking, `.clone` is not required here as dayjs is immutable. I used it, because I noticed this is a codestyle in the dayjs wrapper + it is easier to switch between dayjs and moment implementation.

Should I use `.map()` version without `.clone()` or `for` cycle, what do you think?